flytekit/remote/remote.py

```diff
@@ -473,7 +475,7 @@ def register_launch_plan(
 
         if serialization_settings is None:
             serialization_settings = SerializationSettings(
-                image_config=None, project=self.default_project, domain=self.default_domain
+                image_config=self._config.image_config, project=self.default_project, domain=self.default_domain
```

Comment:
We can leave it None for now... we're only registering a Launch Plan, which we don't need image for, at least for now.